leptonai/api/v1/job.py

```diff
@@ -32,3 +34,34 @@ def update(self, name_or_job: Union[str, LeptonJob], spec: LeptonJob) -> bool:
     def delete(self, name_or_job: Union[str, LeptonJob]) -> bool:
         response = self._delete(f"/jobs/{self._to_name(name_or_job)}")
         return self.ensure_ok(response)
+
+    def get_job_events(self, name_or_job: Union[str, LeptonJob]) -> List[LeptonEvent]:
+        response = self._get(f"/jobs/{self._to_id(name_or_job)}/events")
+        return self.ensure_list(response, LeptonEvent)
+
+    def get_log(
+        self,
+        name_or_job: Union[str, LeptonJob],
+        replica: Union[str, Replica],
+        timeout: Optional[int] = None,
+    ) -> Iterator[str]:
+        """
+        Gets the log of the given deployment's specified replica. The log is streamed
+        in chunks until timeout is reached. If timeout is not specified, the log will be
+        streamed indefinitely, although you should not rely on this behavior as connections
+        can be dropped when streamed for a long time.
+        """
+        replica_id = replica if isinstance(replica, str) else replica.metadata.id_
+        response = self._get(
+            f"/deployments/{self._to_name(name_or_job)}/replicas/{replica_id}/log",
```

Comment:
nit: I assume this should probably be `/jobs/...` instead of `/deployments/...`. See 

https://github.com/leptonai/lepton/blob/main/api-server/httpapi/handler_replica.go#L44-L47

Let's also implement the replicas() function for job?